src/dataframe/dataframe.jl

```diff
@@ -102,6 +103,14 @@ function DataFrame(; kwargs...)
     return result
 end
 
+function DataFrame(pairs::Pair{Symbol}...)
```

Comment:
Looks like you could rewrite the keyword arguments constructor above using the new constructor, right?